LilitYeghiazaryan/recursion_6_7_8_10

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <vector>
+int bigDivider(int num1, int num2) //6
```

Comment:
FYI - the more accurate and common name for this is **gcd** (greatest common divisor)